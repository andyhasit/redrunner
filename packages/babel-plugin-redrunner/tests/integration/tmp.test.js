const {c, EOL, transform} = require('../utils')


test('__clone__ instead of __html__', () => {
  const src = `
  class MyComponent extends Component {
    __clone__ = '<div/>'
  }`
  expect(transform(src)).toMatchSnapshot()
});
