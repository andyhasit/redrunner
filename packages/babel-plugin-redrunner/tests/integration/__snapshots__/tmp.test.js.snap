// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__clone__ instead of __html__ 1`] = `
"class MyComponent extends Component {}

var p = MyComponent.prototype;
p.__ht = '<div></div>';
p.__wc = [];
p.__qc = p.__lu({});
p.__ip = {};

p.__bv = function (component, prototype) {
  component.__bd(prototype, true);

  component.dom = {};
};

p.__cn = undefined;"
`;
